api/v1beta1/zz_generated.deepcopy.go

```diff
@@ -183,21 +184,6 @@ func (in *GitSource) DeepCopy() *GitSource {
 	return out
 }
 
-// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
-func (in *Image) DeepCopyInto(out *Image) {
```

Comment:
Why was this one deleted? :thinking:

Reply:
`Image` struct is used for internal functionality only. Its not a part of any CRD api that why I moved Image struct from `api` pkg to `framework` pkg